src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -237,14 +638,14 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
                   multiline
                   onChange={(ev) => setInfoText(ev.target.value)}
                   rows={4}
-                  sx={{ marginBottom: 2 }}
+                  sx={{ marginBottom: 2, marginLeft: 1, paddingRight: 2 }}
```

Comment:
Same goes here, I see no reason to not just use a consistent margin (and no padding).